packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -110,7 +120,14 @@ export const OptionEditor: React.FunctionComponent<OptionEditorProps> = ({
   ['data-testid']: dataTestId,
   insights,
   disabled = false,
+  savedQueries,
+  onApplyQuery,
+  loadSavedQueries,
 }) => {
+  useEffect(() => {
+    loadSavedQueries();
```

Comment:
We now have two separate places in UI that trigger loading of queries on mount, this is generally not a very conventional way of doing this. Now that you changed the `saveRecentQuery` to reload the action, we can remove `loadSavedQueries` from here and from the `QueryHistoryButtonPopover` and add a dispatch for loading queries in the plugin activate method.